<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Measurement</class>
 <widget class="Applet" name="Measurement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measurement</string>
  </property>
  <property name="windowIcon">
   <iconset resource="project_app.qrc">
    <normaloff>:/board/board.png</normaloff>:/board/board.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="0">
    <widget class="CustomControls::FoldingBox" name="gb_advsettings">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string>Advanced setup settings</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Memory tolerance (cm)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spMemTol">
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="cb_AdAxRg">
          <property name="text">
           <string>Adaptive axis range</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="CustomControls::LineEdit" name="le_S3Y">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="CustomControls::LineEdit" name="le_S3X">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="text">
           <string>60</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sensor 3 coordinates [X, Y] (cm)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="spMMThr">
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Memory threshold (count)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="CustomControls::PushButton" name="pbMeasurement_advsettsave">
          <property name="text">
           <string>Save settings</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Algorithm tolerance</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="spAlgTol">
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="spIntTol">
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Speed of sound (m/s)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="CustomControls::LineEdit" name="le_S2X">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="text">
           <string>30</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string>X </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="CustomControls::ComboBox" name="cb_projection">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>View from behind</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>View from front</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="CustomControls::LineEdit" name="le_S2Y">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>2.25</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="CustomControls::LineEdit" name="le_SOS">
          <property name="text">
           <string>343</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Sensor 2 coordinates [X, Y] (cm)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Type of projection</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Intersecion tolerance</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="CustomControls::PushButton" name="pbMeasurement_uCConfig">
       <property name="text">
        <string>Configure microcontroller</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="CustomControls::ComboBox" name="cbIndirectThr">
       <item>
        <property name="text">
         <string>Thr. 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thr. 2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="CustomControls::PushButton" name="pbMeasurement_getdata">
       <property name="text">
        <string>Get Data</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spInterval">
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>65</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="CustomControls::PushButton" name="pbMeasurement_DefaultCfg">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Default P215 configuration</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;5 ms; 65 ms&gt;; 1 ms step</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Indirect mode THR. table</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Measurement period [ms]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="CustomControls::PushButton" name="pbMeasurement_Once">
       <property name="text">
        <string>Meas. Once</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="CustomControls::PushButton" name="pbMeasurement_Start">
       <property name="text">
        <string>Start/Stop</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomControls::PushButton</class>
   <extends>QPushButton</extends>
   <header>customcontrols.h</header>
  </customwidget>
  <customwidget>
   <class>CustomControls::ComboBox</class>
   <extends>QComboBox</extends>
   <header>customcontrols_combobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Applet</class>
   <extends>QWidget</extends>
   <header>applet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomControls::LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">customcontrols.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomControls::FoldingBox</class>
   <extends>QGroupBox</extends>
   <header>customcontrols_foldingbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="project_app.qrc"/>
 </resources>
 <connections/>
</ui>
